# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lincity-ng/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur lincity-ng-1.0.2-orig/src/lincity-ng/ButtonPanel.hpp lincity-ng-1.0.2/src/lincity-ng/ButtonPanel.hpp
--- lincity-ng-1.0.2-orig/src/lincity-ng/ButtonPanel.hpp	2005-11-14 21:46:51.000000000 +0000
+++ lincity-ng-1.0.2/src/lincity-ng/ButtonPanel.hpp	2006-04-04 15:25:57.873835250 +0000
@@ -62,7 +62,7 @@
   int previousTool;
   int lastShownTechType;
  
-  std::string ButtonPanel::createTooltip( int module, bool root = true );
+  std::string createTooltip( int module, bool root = true );
   std::vector<std::string> mMenuButtons;
   std::vector<std::string> mMenus;
   std::vector<std::string> mButtons;
